includes/profile.php

```diff
@@ -123,7 +133,7 @@ function pmpro_membership_level_profile_fields($user)
          <tr class="more_level_options">
             <th></th>
             <td>
-                <label for="refund_last_subscription"><input value="1" id="refund_last_subscription" name="refund_last_subscription" type="checkbox"> <?php _e("Refund this user's most recent order.", "paid-memberships-pro" ); ?></label>
+                <label for="refund_last_subscription"><input value="1" id="refund_last_subscription" name="refund_last_subscription" type="checkbox" <?php if( !$allows_refunds ) { echo "disabled='true'"; } ?>> <?php _e("Refund this user's most recent order.", "paid-memberships-pro" ); ?></label>
```

Comment:
Disabled is pretty cool for this functionality

```suggestion
                <label for="refund_last_subscription"><input value="1" id="refund_last_subscription" name="refund_last_subscription" type="checkbox"<?php disabled( !$allows_refunds ); ?>> <?php esc_html_e("Refund this user's most recent order.", "paid-memberships-pro" ); ?></label>
```